# Type: cicd
# Subtype: C4
# English Summary: Add image build script
# Chinese Summary: 添加镜像构建脚本
# Entry Number: 12

# === Git Diff Content ===

diff --git a/Dockerfile b/Dockerfile
index 908dc99..0f450ca 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -0,0 +1,10 @@
+FROM node:12
+
+WORKDIR /usr/src/app
+
+COPY ["package.json", "package-lock.json*", "npm-shrinkwrap.json*", "./"]
+
+RUN npm install --silent
+
+COPY . .
+
+EXPOSE 8080